### ISSUE TITLE
webpack failing since highcharts 11.2.0

### ISSUE THREAD

**Issue by thomasleveil** [highcharts/highcharts-vue#252]
Hi,

I updated the dependencies on my project and webpack now fails since highcharts 11.2.0. If I pin down highcharts to 11.1.0, my project builds correctly.

I see in the highcharts changelog that 11.2.0 introduced the following change :
> Upgraded the build pipeline to benefit from new language features. Bundles are now compiled to ES2020 by the TypeScript compiler (*.src.js files), and minified by swc (*.js files).

I wonder if this change is the reason using highcharts-vue in a webpack project now fails.

```


 WAIT  Compiling...                                                                                                                                        14:24:29

98% after emitting CopyPlugin

 ERROR  Failed to compile with 1 error                                                                                                                     14:24:30

 error  in ./node_modules/highcharts/highcharts.js

Module parse failed: Unexpected token (8:5207)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders 
|  * License: www.highcharts.com/license
|  */!function(t,e){"object"==typeof module&&module.exports?(e.default=e,module.exports....
```

**Comment by jakubSzuminski** [comment]
Thanks for reporting @thomasleveil!

I will investigate this issue this week. In the meantime, could you please share more details about your environment?
- which version of Node, npm are you using?
- which version of `highcharts-vue` are you using?
- how are you building your project?
- could you please share your `package.json` config?

Thanks in advance!

**Comment by thomasleveil** [comment]
Here are my versions : 
- node v20.8.1
- highcharts-vue 1.4.3
we are building using vue-cli, so in the end the command is `vue-cli-service serve`.
I cannot share my code as this is not a personal project

**Comment by andrescellini** [comment]
same problem

**Comment by jakubSzuminski** [comment]
Thanks! I'm currently looking into this problem and will try to propose a solution (or at least a workaround) by the end of the next week.

**Comment by jakubSzuminski** [comment]
You pointed out correctly that the most probable cause of this issue is the following update:

> Upgraded the build pipeline to benefit from new language features. Bundles are now compiled to ES2020 by the TypeScript compiler (.src.js files), and minified by swc (.js files).

It seems to me, however, that this issue is unrelated to the `highcharts-vue` package, but more to the `highcharts` package. Either way, I will try to help you solve this issue. Could you please share your webpack config so I can take a look?

It might be the case that your webpack configuration does not support the ES2020 format.

By updating your webpack setup and ensuring it can handle ES2020 features, you should be able to resolve the issue and successfully build your project with Highcharts 11.2.0.

**Comment by SamPulmanCL** [comment]
We are getting the same issue, but we are getting it on all versions of Highcharts 11

**Comment by jakubSzuminski** [comment]
@SamPulmanCL @andrescellini @thomasleveil could you please try to reproduce this issue? It would be great if you could share a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) (like a small .zip) where I could see and investigate this error.

**Comment by thomasleveil** [comment]
Hi @jakubSzuminski, unfortunately the project I'm having the issue on is too big of a mess right now for me to successfully upgrade all the tooling dependencies (webpack and so on). I tried and failed. One day or the other we'll have to pay that price, but our current planning and deadlines won't allow that before 2024Q2.

**Comment by jakubSzuminski** [comment]
Absolutely, I understand. If you cannot update your webpack config then I'd probably advise you to stick with Highcharts 11.1.0 until then.

**Comment by evan7diab** [comment]
same to me, it seems the webpack fails to load highcharts.js

**Comment by jakubSzuminski** [comment]
Could you please reproduce it @evan7diab? We could then investigate this problem and try to propose a solution.

**Comment by alisacorporation** [comment]
2024... still can't use highcharts... by adding "highcharts" to vue.config.js `transpileDependencies: ["vuetify", "highcharts"],` error is gone, but i don't see chart line...

**Comment by alisacorporation** [comment]
maybe it will help someone: (don't ask what here happens - just chatGPT...)  ;)

1. _yarn add -D babel-loader @babel/core @babel/preset-env_

2. **_vue.config.js_**:

`module.exports = {
...
  transpileDependencies: ["vuetify", "highcharts"], // add "highcharts" (in my case i had vuetify before)
...
};
`

3. Clean the Cache

`yarn cache clean`
`rm -rf node_modules`

4. Reinstall Dependencies: 

`yarn install`

5. Start the Development Server:

`yarn serve`
